src/store/modules/stock/actions.ts

```diff
@@ -0,0 +1,63 @@
+import { ActionTree } from 'vuex'
+import store from '@/store'
+import RootState from '@/store/RootState'
+import StockState from './StockState'
+import * as types from './mutation-types'
+
+const actions: ActionTree<StockState, RootState> = {
+  async updatedStockList ({commit, rootGetters}, items) {
+    const productIds = items.map((item: any) => item.shopifyProductSKU)
+    const facilityIds = [...new Set(items.map((item: any) => item.externalFacilityId))]
+    const viewSize = productIds.length;
+    const viewIndex = 0;
+    const payload = {
+      viewSize,
+      viewIndex,
+      productIds
+    }
+    await store.dispatch('user/fetchFacilityLocations', facilityIds);
+    await store.dispatch("product/fetchProducts", payload);
+    const unidentifiedItems = [] as any;
+    const parsed = items.map((item: any) => {
+      const product = rootGetters['product/getProduct'](item.shopifyProductSKU)
```

Comment:
We should not use getter here, instead get products list with product/fetchProducts action and then use that